File: backend/internal/models/storage.go
package models

import "time"

type Storage struct {
	ID        string        `json:"id"`
	JKID      string        `json:"jk_id"`
	Number    string        `json:"number"`
	Area      float64       `json:"area"`
	Floor     int           `json:"floor"`
	Price     float64       `json:"price"`
	Entrance  int           `json:"entrance"`
	Status    StorageStatus `json:"status"`
	CreatedAt time.Time     `json:"created_at"`
}

File: frontend/src/app/core/models/storage.ts
export type StorageStatus = 'free' | 'occupied' | 'locked';

export interface Storage {
  id: string;
  jk_id: string;
  number: string;
  area: number;
  floor: number;
  entrance: number;
  status: StorageStatus;
  created_at: string;
}

export interface CreateStorageRequest {
  jk_id?: string;
  number: string;
  area: number;
  floor: number;
  entrance: number;
}

export interface UpdateStorageRequest {
  number: string;
  area: number;
  floor: number;
  entrance: number;
  status: StorageStatus;
}
